pkg/agent/bakerapi.go

```diff
@@ -23,7 +23,7 @@ func NewAgentBaker() (AgentBaker, error) {
 
 type agentBakerImpl struct{}
 
-func (agentBaker *agentBakerImpl) GetNodeBootstrapping(ctx context.Context,
+func (agentBaker *agentBakerImpl) GetNodeBootstrapping(_ context.Context,
```

Comment:
lol, we probably should be using ctx somehow...

Reply:
hmmmmm https://github.com/Azure/AgentBaker/blob/10134a39958e3ee9a04e76644c1a0632d163ae99/apiserver/getnodebootstrapdata.go#L23C1-L25C16

Reply:
Should we be logging the ctx or is it not being passed to anything atm when it should be?

Reply:
honestly I think it would be best to leave `ctx` as it is and add a TODO note to figure out what to do with it, as it stands the timeout we set in the calling func isn't having any effect when it really should be...

I'd be open to further discussion around this, maybe we legitimately don't need it (which would kinda make sense since GetNodeBootstrapping doesn't make any outbound RPCs or anything)

Reply:
@anujmaheshwari1 @ganesshkumar @UtheMan